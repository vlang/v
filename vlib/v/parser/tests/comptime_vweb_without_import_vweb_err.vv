fn main() {
	$vweb.html('index.html')
}
