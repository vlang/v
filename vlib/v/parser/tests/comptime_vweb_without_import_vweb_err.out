vlib/v/parser/tests/comptime_vweb_without_import_vweb_err.vv:2:2: error: `$vweb` cannot be used without importing vweb
    1 | fn main() {
    2 |     $vweb.html('index.html')
      |     ~~~~~
    3 | }
